docs/source/whats_new.rst

```diff
@@ -10,16 +10,22 @@ For a full list of included Pull Requests and closed Issues, please see the
 Features
 --------
 
-* Kevin Donkers and Phil Elson made the AdaptiveScalar to the Natural
-  Earth Features the default. This will make the default features look
-  much nicer when plotting on zoomed in axes. (:pull:`1105`)
+* We are very pleased to announce that Greg Lucas has been added to the cartopy
```

Comment:
Should `Cartopy` be capitalized? I see below it is actually listed as `CartoPy`.

Reply:
I do prefer to capitalize, but we aren't particularly consistent about it, it seems.